swagger/src/test/scala/me/apidoc/swagger/SwaggerServiceValidatorSpec.scala

```diff
@@ -111,6 +101,16 @@ class SwaggerServiceValidatorSpec extends FunSpec with Matchers {
                       `type` = "long",
                       required = true
                     ),
+                    Field(
```

Comment:
Should we keep the original order in `json` ?
Would it be an unexpected behavior if generated code have different order than originally specified in `json`?

Reply:
objects in JSON are unordered, so the original code is not any better than what I have here

Model.fields is a Seq, but it should really be a Set. I don't think the fix is trivial, so I just wanted the tests to pass for now.